ts/sagas/wallet.ts

```diff
@@ -1019,6 +1037,15 @@ function* completionHandler(
   }
 }
 
+export function* loadWalletHome(pagoPaClient: PagoPaClient) {
+  yield put(walletHomeLoadRequest());
```

Comment:
I'm not sure the "wallet home" request/success actions make sense, since they never could fail and there is not a request to a "home wallet" resource.

I haven't checked but I believe we already have wallets and transactions requests related actions.